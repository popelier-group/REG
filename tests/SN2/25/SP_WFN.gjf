# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -1.36911600   -0.00005000    0.00008100
 H                 -1.04650900   -0.87587100   -0.54573600
 H                 -1.04647800    0.91054900   -0.48548100
 H                 -1.04656800   -0.03484200    1.03149000
 Br                 1.67926400    0.00000900   -0.00000800
 Cl                -3.22948000    0.00000200   -0.00001800

frame.wfn
