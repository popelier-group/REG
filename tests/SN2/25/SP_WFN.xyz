
6

C       -1.245824    0.001372    0.040230
H       -0.895718   -0.544809    0.905196
H       -0.953212   -0.484494   -0.880296
H       -0.921251    1.032444    0.064606
Br        1.800968   -0.001881   -0.058098
Cl       -3.105222    0.003203    0.100150
