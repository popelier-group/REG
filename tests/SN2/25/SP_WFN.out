 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.25/Gau-19349.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.25/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     19350.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -1.36912  -0.00005   0.00008 
 H                    -1.04651  -0.87587  -0.54574 
 H                    -1.04648   0.91055  -0.48548 
 H                    -1.04657  -0.03484   1.03149 
 Br                    1.67926   0.00001  -0.00001 
 Cl                   -3.22948   0.       -0.00002 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.369116   -0.000050    0.000081
      2          1           0       -1.046509   -0.875871   -0.545736
      3          1           0       -1.046478    0.910549   -0.485481
      4          1           0       -1.046568   -0.034842    1.031490
      5         35           0        1.679264    0.000009   -0.000008
      6         17           0       -3.229480    0.000002   -0.000018
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.081228   0.000000
     3  H    1.081229   1.787436   0.000000
     4  H    1.081227   1.787448   1.787447   0.000000
     5  Br   3.048380   2.914588   2.914522   2.914681   0.000000
     6  Cl   1.860364   2.414606   2.414595   2.414607   4.908744
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.245824    0.001372    0.040230
      2          1           0       -0.895718   -0.544809    0.905196
      3          1           0       -0.953212   -0.484494   -0.880296
      4          1           0       -0.921251    1.032444    0.064606
      5         35           0        1.800968   -0.001881   -0.058098
      6         17           0       -3.105222    0.003203    0.100150
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         156.9521758           0.8221982           0.8221982
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       169.5496817562 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  2.62D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.40135813     A.U. after   11 cycles
            NFock= 11  Conv=0.69D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.55572-101.40673 -62.20880 -56.02660 -56.02653
 Alpha  occ. eigenvalues --  -56.02653 -10.08036  -9.32106  -8.41758  -7.08470
 Alpha  occ. eigenvalues --   -7.07633  -7.07633  -6.24066  -6.24029  -6.24029
 Alpha  occ. eigenvalues --   -2.35305  -2.35292  -2.35292  -2.35262  -2.35262
 Alpha  occ. eigenvalues --   -0.70650  -0.54155  -0.48891  -0.30531  -0.30530
 Alpha  occ. eigenvalues --   -0.26926  -0.17326  -0.17325  -0.05420  -0.05353
 Alpha  occ. eigenvalues --   -0.05353
 Alpha virt. eigenvalues --    0.11047   0.12531   0.14927   0.14927   0.17495
 Alpha virt. eigenvalues --    0.18500   0.19591   0.19591   0.20768   0.20768
 Alpha virt. eigenvalues --    0.21753   0.22547   0.22547   0.24843   0.28518
 Alpha virt. eigenvalues --    0.30904   0.30904   0.36915   0.40572   0.53696
 Alpha virt. eigenvalues --    0.53697   0.55463   0.58677   0.58678   0.59754
 Alpha virt. eigenvalues --    0.62087   0.62087   0.65654   0.76770   0.76770
 Alpha virt. eigenvalues --    0.79541   0.84305   0.87838   0.87838   0.89189
 Alpha virt. eigenvalues --    0.89189   0.95586   0.97128   0.97128   1.01349
 Alpha virt. eigenvalues --    1.01350   1.03831   1.10838   1.26680   1.26681
 Alpha virt. eigenvalues --    1.26989   1.57072   1.67848   1.67848   1.85431
 Alpha virt. eigenvalues --    1.85431   1.87936   2.08801   2.21278   2.21279
 Alpha virt. eigenvalues --    2.27848   2.27848   2.30688   2.45820   2.45820
 Alpha virt. eigenvalues --    2.57723   2.59951   2.64253   2.64253   2.67530
 Alpha virt. eigenvalues --    2.67530   3.12882   3.29137   3.87396   3.87397
 Alpha virt. eigenvalues --    4.39584   4.39585   4.40423   4.40424   4.53354
 Alpha virt. eigenvalues --    6.76783   7.80357   7.80361   7.86522   9.84304
 Alpha virt. eigenvalues --   24.03747  25.99025  25.99025  26.49537  48.08107
 Alpha virt. eigenvalues --  215.82888 290.01218 290.01218 290.068271020.84013
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.692423   0.406083   0.406078   0.406084  -0.064543  -0.189883
     2  H    0.406083   0.521106  -0.038534  -0.038536  -0.000984  -0.042131
     3  H    0.406078  -0.038534   0.521102  -0.038534  -0.000988  -0.042125
     4  H    0.406084  -0.038536  -0.038534   0.521117  -0.000980  -0.042138
     5  Br  -0.064543  -0.000984  -0.000988  -0.000980  35.924438   0.059384
     6  Cl  -0.189883  -0.042131  -0.042125  -0.042138   0.059384  17.263311
 Mulliken charges:
               1
     1  C   -0.656240
     2  H    0.192996
     3  H    0.193001
     4  H    0.192988
     5  Br  -0.916327
     6  Cl  -0.006419
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C   -0.077255
     5  Br  -0.916327
     6  Cl  -0.006419
 Electronic spatial extent (au):  <R**2>=           1167.7204
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             -4.2133    Y=              0.0045    Z=              0.1360  Tot=              4.2155
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -75.3680   YY=            -44.3997   ZZ=            -44.4319
   XY=              0.0321   XZ=              0.9989   YZ=             -0.0010
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -20.6348   YY=             10.3335   ZZ=             10.3013
   XY=              0.0321   XZ=              0.9989   YZ=             -0.0010
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=             46.7293  YYY=              0.8403  ZZZ=             -0.9229  XYY=              8.6258
  XXY=             -0.0315  XXZ=             -0.9512  XZZ=              8.6401  YZZ=             -0.8752
  YYZ=             -0.1901  XYZ=             -0.0271
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=          -1564.0837 YYYY=            -75.9671 ZZZZ=            -77.3784 XXXY=              0.8508
 XXXZ=             26.2983 YYYX=             -0.0118 YYYZ=              0.0007 ZZZX=             21.7809
 ZZZY=             -0.0914 XXYY=           -249.6182 XXZZ=           -249.9872 YYZZ=            -25.5535
 XXYZ=              0.0334 YYXZ=              7.1626 ZZXY=              0.9421
 N-N= 1.695496817562D+02 E-N=-7.692521643003D+03  KE= 3.070054355269D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE557\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\V69787FF\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-1.369116,-0.00005,0.000081\H,0,-1.046509,-0.875871,-0.545736\H,0,
 -1.046478,0.910549,-0.485481\H,0,-1.046568,-0.034842,1.03149\Br,0,1.67
 9264,0.000009,-0.000008\Cl,0,-3.22948,0.000002,-0.000018\\Version=ES64
 L-G16RevC.01\State=1-A\HF=-3074.4013581\RMSD=6.862e-09\Dipole=-1.65851
 26,-0.0000399,0.0000617\Quadrupole=-15.365474,7.6827314,7.6827426,-0.0
 000291,-0.0000695,-0.00001\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 ALL THINGS ARE POSSIBLE EXCEPT SKIING THROUGH A REVOLVING DOOR.
 Job cpu time:       0 days  0 hours  0 minutes 29.4 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 15.3 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:53:26 2020.
